content/messaging/agent-chat-api/rtm-reference/index.mdx

```diff
@@ -3464,6 +3460,22 @@ No response payload (`200 OK`).
 
 </CodeSample>
 
+<CodeResponse>
+
+```json
+{
+  "request_id": "<request_id>", // optional
```

Comment:
```suggestion
  "request_id": "<request_id>", 
```

Reply:
I try to get rid of those "JSON" comments when possible. They'll be deleted anyway in the near future.